utils/index.js

```diff
@@ -1,11 +1,12 @@
 const lzString = require('lz-string');
 
-const compressParams = ({ code, themes, widths, theme }) => {
+const compressParams = ({ code, themes, widths, theme, title }) => {
   const data = JSON.stringify({
     ...(code ? { code } : {}),
     ...(themes ? { themes } : {}),
     ...(widths ? { widths } : {}),
     ...(theme ? { theme } : {}),
+    ...(title ? { title } : {}),
```

Comment:
Noticed we are not passing the `title` via the `createUrl` function below. This could be a cool addition allow URLs to be generated with pre-defined titles.

Reply:
Good idea, should be added now!